solr/core/src/java/org/apache/solr/response/RawResponseWriter.java

```diff
@@ -117,6 +119,10 @@ public void write(OutputStream out, SolrQueryRequest request, SolrQueryResponse
       try (InputStream in = content.getStream()) {
         in.transferTo(out);
       }
+    } else if (obj != null && (obj instanceof SolrCore.RawWriter)) {
+      final var rawWriter = (SolrCore.RawWriter) obj;
+      rawWriter.write(out);
+      if (rawWriter instanceof Closeable) ((Closeable) rawWriter).close();
```

Comment:
maybe a `{` and `}`?

Reply:
This is something that carried over from existing code.

I've agreed to a few variable renames above in code that preexists this PR, but I'll probably skip this change.  IMO this code could use lots of reformatting/restructuring attention, but we need to draw the line somewhere.